src/NLog/Internal/StringBuilderExt.cs

```diff
@@ -150,14 +150,18 @@ public static void CopyToStream(this StringBuilder builder, MemoryStream ms, Enc
 #if !SILVERLIGHT
             if (transformBuffer != null)
             {
+                int charCount = 0, byteCount = 0;
```

Comment:
please move the bytecount to a new declaration (consistent code style)

Reply:
Fixed